docs/podman-container-runlabel.1.md

```diff
@@ -85,6 +89,11 @@ Print usage statement
 
 Suppress output information when pulling images
 
+**--replace**
+
+If a container with the same name exists in local storage, it will be removed prior
+to executing the label command.
+
```

Comment:
I know it's pre-existing, but if you get other comments, could you move --display down below the '--c*' options please?